src/HylianShield/Validator/String/Subset.php

```diff
@@ -44,6 +44,17 @@
     protected $ranges = array();
 
     /**
+     * Check the length of the given string.
+     *
+     * @param string $string
+     * @return int
+     */
+    public function checkLength($string)
+    {
+        return mb_strlen($string, static::ENCODING);
```

Comment:
The last part of this line is where it all went *boom* in the previous commit @CyberSecutor.
Your length check would have failed in production.